pkgs/development/libraries/gegl/4.0.nix

```diff
@@ -120,8 +120,8 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "Graph-based image processing framework";
-    homepage = "http://www.gegl.org";
-    license = licenses.gpl3;
+    homepage = "https://www.gegl.org";
+    license = licenses.gpl3Plus;
```

Comment:
Looks like it is actually `lgpl3Plus`: https://gitlab.gnome.org/GNOME/gegl/-/commit/6609e9b8e1a5f8fd8d06de6109f45a8e43c4f6a3